<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover {
	background-color: #ffdbcc;
}

QPushButton:pressed {
	background-color: #fcb9aa;
}
QPushButton {
	background-color: rgb(57, 208, 208);
	border-radius: 31px;
}</string>
    </property>
    <property name="text">
     <string>Изменить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_input">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Введите новое имя товара</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="price_input">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Введите цену товара</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="count_input">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Введите количество товара</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="path_input">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Введите путь изображения</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
